packages/bsky/src/api/blob-resolver.ts

```diff
@@ -83,20 +83,16 @@ export async function resolveBlob(
   db: Database,
   idResolver: IdResolver,
 ) {
+  const { ref } = db.db.dynamic
   const cidStr = cid.toString()
+
   const [{ pds }, takedown] = await Promise.all([
     idResolver.did.resolveAtprotoData(did), // @TODO cache did info
     db.db
-      .selectFrom('moderation_action_subject_blob')
-      .select('actionId')
-      .innerJoin(
-        'moderation_action',
-        'moderation_action.id',
-        'moderation_action_subject_blob.actionId',
-      )
-      .where('cid', '=', cidStr)
-      .where('action', '=', TAKEDOWN)
-      .where('reversedAt', 'is', null)
+      .selectFrom('moderation_subject_status')
+      .select('id')
+      .where(sql<string>`${ref('blobCids')} @> ${JSON.stringify([cidStr])}`)
```

Comment:
I think we may be able to take more advantage of kysely here since it knows about the `@>` operator.
```ts
      .where('blobCids', '@>', json([cidStr]))
```

Where `json` is a helper like:
```ts
function json<T>(value: T) {
  return sql`CAST(${JSON.stringify(value)} AS JSONB)`
}
```

Reply:
ah TIL!
avoiding the helper for now and using direct sql helper.